apps/concierge_site/lib/mail_templates/notification.html.eex

```diff
@@ -344,5 +344,6 @@
 
       </div>
 
+      <img src="<%= track_open_url %>" height=0 width=0>
```

Comment:
We should also add this to the original MJML template so we don't forget about it when we do that cleanup.

Reply:
ah, good call out 👍 thanks!